client/python/lib/README.md

```diff
@@ -40,3 +44,82 @@ Apart from `make build`, there are also other targets available:
    `make test PACKAGE_PATH=/opt/packages/ovmsclient-0.1-py3-none-any.whl`
 
  - `make clean` - removes all intermediate files generated while building the package
+
+
+## Usage
+
+**Create gRPC client instance:**
+```python
+import ovmsclient
+
+config = {
+   "address": "localhost", 
+   "port": 9000
+   }
+
+client = ovmsclient.make_grpc_client(config="config")
+```
+
+**Create and send model status request:**
+```python
+status_request = ovmsclient.make_grpc_status_request(model_name="model")
+status_response = client.get_model_status(status_request)
+status_response.to_dict()
+
+# Examplary status_response.to_dict() output:
+#
+# {
+#    "1": {
+#        "state": "AVAILABLE", 
+#        "error_code": 0, 
+#        "error_message": ""
+#    }             
+# } 
+#
+```
+
+**Create and send model metadata request:**
+```python
+metadata_request = ovmsclient.make_grpc_metadata_request(model_name="model")
+metadata_response = client.get_model_metadata(metadata_request)
+metadata_response.to_dict()
+
+# Examplary metadata_response.to_dict() output:
+#
+#{
+#   "1": {
+#       "inputs": {
+#           "input": {
+#               "shape": (1, 3, 224, 224),
+#               "dtype": DT_FLOAT32  
+#           }
+#       },
+#       "outputs" {
+#           "output": {
+#               "shape": (1, 1000),
+#               "dtype": DT_FLOAT32  
+#           }
+#       }
+#   }
+#}
+#
+```
+
+**Create and send predict request with binary input data:**
+```python
+with open(<path_to_img>, 'rb') as f:
+    img = f.read()
+predict_request = ovmsclient.make_grpc_predict_request(
+    { "input": img }, model_name="model")
+predict_response = client.predict(predict_request)
+predict_response.to_dict()
+
+# Examplary predict_response.to_dict() output:
+#
+#{
+#   "output": [[0.01, 0.03, 0.91, ... , 0.00021]]
+#}
+#
+```
```

Comment:
Maybe instead of input/output we could place input_name/output_name? What do you think?